src/main/java/models/Customer.java

```diff
@@ -77,17 +85,17 @@ public void setPhoneNumber(String phoneNumber) {
         this.phoneNumber = phoneNumber;
     }
 
-    @OneToOne(fetch = FetchType.EAGER, cascade=ALL)
+    @OneToOne( cascade=ALL)
     public ShoppingCart getShoppingCart(){
         return this.shoppingCart;
     }
     public void setShoppingCart(ShoppingCart shoppingCart){
         this.shoppingCart = shoppingCart;
     }
 
-    @OneToMany(fetch = FetchType.EAGER, cascade=ALL, mappedBy="customer")
```

Comment:
Any particular reason that fetch is no longer necessary?

Reply:
EAGER fetch causes problems loading the DB sometimes, and it is unnecessary